### ISSUE TITLE
Unable to configure airflow host

### ISSUE THREAD

**Issue by kfirgollan** [necatiarslan/airflow-vscode-extension#15]
When I try to select a host to connect to I get `undefined`
![image](https://user-images.githubusercontent.com/155785/208898030-97155bb1-c8ab-49cd-a27d-ba471b7d2167.png)

When adding additional hosts the `undefined` is still there and I can't connect to the other one
![image](https://user-images.githubusercontent.com/155785/208898369-5b7d8c82-4ac9-4029-90d4-c884a9cc5f2b.png)

No logs or any error indication are available in vscode

Note: I used a local airflow instance based on the docker-compose from the airflow repo ([here](https://github.dev/apache/airflow/blob/74e82af7eefe1d0d5aa6ea1637d096e4728dea1f/docs/apache-airflow/howto/docker-compose/docker-compose.yaml#L1))

VSCode details
![image](https://user-images.githubusercontent.com/155785/208898711-ba36d80f-54d4-44fd-a892-8f7315a574c0.png)

**Comment by necatiarslan** [comment]
Thank you for using our extension and for providing feedback. I am currently trying to investigate the issue you mentioned, but we haven't been able to reproduce it yet. In order to better understand the problem, could you please provide more details on the steps that led to the invalid connection? Here are some specific questions to help guide you:

- Can you tell me exactly what actions you took before the buggy connections appeared?
- Were you able to delete the undefined connection?
- Did the undefined connection appear after you reopened VSCode or the extension?
- Does the invalid connection still appear after you reopen VSCode?
- Could you send me the logs from the "Airflow-Log" section of the VSCode Output after you delete the invalid connection and then see it reappear?

**Comment by kfirgollan** [comment]
Thanks for the explanations :)

So I went ahead and removed the folder you provided on both ends (the windows and the remote machine).
Removed the extension and installed it again -> still got the same result.

The logs
![image](https://user-images.githubusercontent.com/155785/209175883-186206bd-5ffa-429e-8327-860a061b9c47.png)

I would note that the even after the removal of the extension I had a different entry that stayed there as well (even after I deleted the mentioned folder)
![image](https://user-images.githubusercontent.com/155785/209176236-0addf56c-0daf-48c7-b116-eef92dec6bb8.png)
Perhaps there is another location that needs to be cleared?
Again note the remote execution mode here, perhaps additional folders ?
